docs/build.gradle

```diff
@@ -6,10 +6,10 @@ plugins {
 
 // 2. Include Orchid dependencies
 dependencies {
-    orchidRuntime "io.github.javaeden.orchid:OrchidDocs:0.21.0"
-    orchidRuntime "io.github.javaeden.orchid:OrchidKotlindoc:0.21.0"
-    orchidRuntime "io.github.javaeden.orchid:OrchidPluginDocs:0.21.0"
-    orchidRuntime("io.github.javaeden.orchid:OrchidEditorial:0.21.0")
+//    orchidRuntime "io.github.javaeden.orchid:OrchidDocs:0.21.0"
```

Comment:
remove if not needed?

Reply:
It's needed for the docs but the package is dead and cant be used anymore, keeping it only for reference when I'll get to fix it.